trading_calendars/calendar_utils.py

```diff
@@ -4,42 +4,43 @@
     CyclicCalendarAlias,
     InvalidCalendarName,
 )
-from .exchange_calendar_bmf import BMFExchangeCalendar
-from .exchange_calendar_cfe import CFEExchangeCalendar
-from .exchange_calendar_cme import CMEExchangeCalendar
-from .exchange_calendar_fwb import FWBExchangeCalendar
-from .exchange_calendar_ice import ICEExchangeCalendar
-from .exchange_calendar_lse import LSEExchangeCalendar
-from .exchange_calendar_nyse import NYSEExchangeCalendar
-from .exchange_calendar_tse import TSEExchangeCalendar
-from .exchange_calendar_tsx import TSXExchangeCalendar
+from .exchange_calendar_bvmf import BVMFExchangeCalendar
+from .exchange_calendar_xcbf import XCBFExchangeCalendar
+from .exchange_calendar_cmes import CMESExchangeCalendar
+from .exchange_calendar_xfra import XFRAExchangeCalendar
+from .exchange_calendar_iepa import IEPAExchangeCalendar
+from .exchange_calendar_xlon import XLONExchangeCalendar
+from .exchange_calendar_xnys import XNYSExchangeCalendar
+from .exchange_calendar_xtks import XTKSExchangeCalendar
+from .exchange_calendar_xtse import XTSEExchangeCalendar
 from .us_futures_calendar import (
     QuantopianUSFuturesCalendar,
 )
 from .weekday_calendar import WeekdayCalendar
 
 _default_calendar_factories = {
-    'NYSE': NYSEExchangeCalendar,
-    'CME': CMEExchangeCalendar,
-    'FWB': FWBExchangeCalendar,
-    'ICE': ICEExchangeCalendar,
-    'CFE': CFEExchangeCalendar,
-    'BMF': BMFExchangeCalendar,
-    'LSE': LSEExchangeCalendar,
-    'TSX': TSXExchangeCalendar,
-    'TSE': TSEExchangeCalendar,
+    'XNYS': XNYSExchangeCalendar,
```

Comment:
For backwards compat, I think we should restore aliases for the calendars we've already released (I think that's NYSE, CME, ICE, CFE, BMF, LSE, and TSX). Luckily, all of these except NYSE are 3 characters, so there's no risk of collision moving forward. Grepping internally, I see usages of `get_calendar` with `CME`, `ICE`, and `CFE`, so I think we depend on at least those.

In the future, we may need to change the ICE alias, since XICE is the official code for the NASDAQ Iceland, which means we probably have conflicting vendor data with an exchange code of ICE.